tests/samples/grundsteuer_sample_input.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+grundsteuer_sample_input = {
+    "eigentuemer": {
+        "person": [
+            {"persoenlicheAngaben": {
+                "anrede": "frau",
+                "titel": "Dr",
+                "name": "Granger",
+                "vorname": "Hermine",
+                "geburtsdatum": datetime.date(1979, 9, 19)},
```

Comment:
This is to produce whatever datetime format the parser is expecting at the other end, right? Let's use the actual string value here (e.g. "1979-09-19") so that this would emulate what the actual incoming json would look like as closely as possible. If we end up using another date format, we would then modify this value to reflect the new format.